### PR TITLE
2024 H2 Cohort Website Updates

### PR DESCRIPTION
**Pull request by daaimah123**
Updated the FT page to be inclusive of the July 2024 5-month cohort and temporarily removed the info video.

### Apply Buttons
Top of the FT page has been updated to include the short application form. 
apply now button top of FT page
Middle of the FT page has been updated to include the short application form. 
apply now button middle of FT page
Top of homepage has been updated to route to FT page. 
apply now button on homepage

### Mentor Application Timeline
Has been updated to highlight the accelerated application process unique to this cohort's application process.
Screenshot 2024-05-02 at 8 09 47 AM

### Tuition Content Update
Screenshot 2024-05-02 at 8 11 08 AM

### CODE REVIEW

**Comment by daaimah123** [comment]
### Note: I've clarified [in [13eb609](https://github.com/Techtonica/techtonica.org/pull/336/commits/13eb609da81466321444434eb44e0d054292c765)] the application and interview period to be the same from March 1st - June 7th.

**Review by sunnybrie — COMMENTED** [review]
Just a little typo but it won't let me directly fix it for some reason.

File: templates/full-time-program.html [hunk 1]
```diff
@@ -7,7 +7,7 @@ <h1 class="main-header__header">Full-Time Program</h1>
       <p>
         Six months of free, intensive, hands-on software engineering learning,
         followed by six months of remote job placement or job search support for women and
-        non-binary adults seeking economic empowerment.
+        non-binary adults seeking economic empowerment. The July 2024 cohort will be a five month cohort.
```
Comment (MichelleGlauser) [inline templates/full-time-program.html:10]: five-month

File: templates/full-time-program.html [hunk 2]
```diff
@@ -18,8 +18,8 @@ <h2>
     <div class="column">
       <h1 class="blue-h1">The Program</h1>
       <p class="justify">
-        Techtonica's full-time software engineering training program is a free,
-        virtual, year-long program that provides the opportunity for
+        Techtonica's full-time software engineering training program is a 
+        virtual, close to year-long program that provides the opportunity for
```
Comment (MichelleGlauser) [inline templates/full-time-program.html:22]: “Techtonica’s full-time software engineering training program is a virtual, close to year-long program that provides the opportunity for” —> “Techtonica provides the opportunity for”

File: templates/full-time-program.html [hunk 3]
```diff
@@ -52,8 +52,9 @@ <h1 class="blue-h1">The Program</h1>
       </div>
       <p class="justify">
         The program usually runs Monday through Friday, 8:00 AM to 5:00 PM (Pacific Time)
-        for six months (from January to June and July to December). The whole program is
-        a year long, starting with training at Techtonica, and followed by six months of
+        for six months (from January to June and July to December). The July 2024 cohort will
+        be a five month cohor, running from July to November. The whole program is
+        about a year long, starting with training at Techtonica, and followed by six months of
```
Comment (MichelleGlauser) [inline templates/full-time-program.html:57]: "The whole program is about a year long, starting with training at Techtonica, and followed by six months of" —> "The training is followed by six months of"

File: templates/full-time-program.html [hunk 4]
```diff
@@ -189,9 +189,10 @@ <h1>Eligibility</h1>
         <li>are willing to be part of the program during Pacific Time hours</li>
         <li>are dedicated to building a long-term career in tech and contributing to a diverse, inclusive, equitable
           tech environment</li>
-        <li>have less than the amount specified in your location and situation on the <a
+        <!-- In light of the tuition change, do we need to change the wording for this bullet?  -->
+        <!-- <li>have less than the amount specified in your location and situation on the <a
```
Comment (MichelleGlauser) [inline templates/full-time-program.html:193]: Just remove this as we mention elsewhere how this makes people eligible to apply for scholarships and stipends, right?

File: templates/full-time-program.html [hunk 5]
```diff
@@ -224,11 +225,13 @@ <h1>Eligibility</h1>
...
         </li>
       </ul>
+      <!-- Do we need to say anything about the Fall 2023 application process here? -->
       <p class="justify">If you don't get in the first time, you can apply two more times for a maximum of three times,
-        and they do not have to be consecutive application rounds. If you have any special circumstances that make you
-        question your eligibility for Techtonica’s
-        program, please feel free to contact us and we will be happy to discuss with you. If you are interested in
-        Techtonica but cannot participate in the next cohort, please fill out our
+        and they do not have to be consecutive application rounds. Fall 2023 applicants that passed their final round of 
+        the incomplete Fall 2023 application process may receive invitation to continue participating the July 2024
```
Comment (MichelleGlauser) [inline templates/full-time-program.html:225]: "may receive invitation" — "will be invited"
"participating the" —> "participating in the"

File: templates/full-time-program.html [hunk 6]
```diff
@@ -267,9 +270,10 @@ <h1 class="blue-h1">Application Process</h1>
       For the January to June ("H1") cohort, the application forms open, emails to
       interested folks go out, and forms close again around September.
       For the July to December ("H2") cohort, the application forms open,
-      emails to interested folks go out, and forms close again around March.
-      The application process continues for about five weeks after applications
-      close, with candidates eliminated at each step of the process.
+      emails to interested folks go out, and forms close again around March. 
+      The July 2024 cohort interest emails will go out in April and the application
```
Comment (MichelleGlauser) [inline templates/full-time-program.html:268]: For the 2024 H2 cohort, applications open in April and close in May.

File: templates/full-time-program.html (hunk 4 above)
Comment (daaimah123) [reply templates/full-time-program.html]: ### ✅ Addressed in [1e45bab](https://github.com/Techtonica/techtonica.org/pull/336/commits/1e45bab21cda780bd785d464f8c358bf46a9f5c1).

File: templates/full-time-program.html [hunk 7]
```diff
@@ -53,8 +52,9 @@ <h1 class="blue-h1">The Program</h1>
       </div>
       <p class="justify">
         The program usually runs Monday through Friday, 8:00 AM to 5:00 PM (Pacific Time)
-        for six months (from January to June and July to December). The whole program is
-        a year long, starting with training at Techtonica, and followed by six months of
+        for six months (from January to June and July to December). The July 2024 cohort will
+        be a five month cohor, running from July to November. The whole program is
```
[the remark this answers is not in the document]
Comment (daaimah123) [reply templates/full-time-program.html]: ### ✅ Addressed in [1e45bab](https://github.com/Techtonica/techtonica.org/pull/336/commits/1e45bab21cda780bd785d464f8c358bf46a9f5c1).

File: templates/full-time-program.html (hunk 6 above)
Comment (daaimah123) [reply templates/full-time-program.html]: ### ✅ Addressed in [476c912](https://github.com/Techtonica/techtonica.org/pull/336/commits/476c912449d5aed0a1343b11235d47ed52ed07db).

File: templates/full-time-program.html (hunk 5 above)
Comment (daaimah123) [reply templates/full-time-program.html]: ### ✅ Addressed in [476c912](https://github.com/Techtonica/techtonica.org/pull/336/commits/476c912449d5aed0a1343b11235d47ed52ed07db).

File: templates/full-time-program.html [hunk 8]
```diff
@@ -224,11 +225,13 @@ <h1>Eligibility</h1>
         <li>have attended at least one of the study night groups for the application process
         </li>
       </ul>
+      <!-- Do we need to say anything about the Fall 2023 application process here? -->
       <p class="justify">If you don't get in the first time, you can apply two more times for a maximum of three times,
-        and they do not have to be consecutive application rounds. If you have any special circumstances that make you
-        question your eligibility for Techtonica’s
-        program, please feel free to contact us and we will be happy to discuss with you. If you are interested in
-        Techtonica but cannot participate in the next cohort, please fill out our
+        and they do not have to be consecutive application rounds. Fall 2023 applicants that passed their final round of
```
[the remark this answers is not in the document]
Comment (daaimah123) [reply templates/full-time-program.html]: ### ✅ Addressed in [476c912](https://github.com/Techtonica/techtonica.org/pull/336/commits/476c912449d5aed0a1343b11235d47ed52ed07db).

File: templates/full-time-program.html (hunk 2 above)
Comment (daaimah123) [reply templates/full-time-program.html]: ### ✅ Addressed in [1e45bab](https://github.com/Techtonica/techtonica.org/pull/336/commits/1e45bab21cda780bd785d464f8c358bf46a9f5c1).

File: templates/full-time-program.html (hunk 1 above)
Comment (daaimah123) [reply templates/full-time-program.html]: ### ✅ Addressed in [1e45bab](https://github.com/Techtonica/techtonica.org/pull/336/commits/1e45bab21cda780bd785d464f8c358bf46a9f5c1).

File: templates/full-time-program.html (hunk 3 above)
Comment (daaimah123) [reply templates/full-time-program.html]: ### ✅ Addressed in [1e45bab](https://github.com/Techtonica/techtonica.org/pull/336/commits/1e45bab21cda780bd785d464f8c358bf46a9f5c1).

File: templates/full-time-program.html [hunk 9]
```diff
@@ -52,8 +52,9 @@ <h1 class="blue-h1">The Program</h1>
       </div>
       <p class="justify">
         The program usually runs Monday through Friday, 8:00 AM to 5:00 PM (Pacific Time)
-        for six months (from January to June and July to December). The whole program is
-        a year long, starting with training at Techtonica, and followed by six months of
+        for six months (from January to June and July to December). The July 2024 cohort will
+        be a five month cohor, running from July to November. The whole program is
+        about a year long, starting with training at Techtonica, and followed by six months of
         remote job placement or job search and community support. After the first trial month of
         full-time training, participants who work the hardest and meet and exceed
```
Comment (MichelleGlauser) [inline templates/full-time-program.html]: After "support" is where we should put the three tuition-related paragraphs from G2, G3, and G4. And let's just remove the "After the first trial month of full-time learning" and start with "Participants who work hard and meet and exceed expectations will be allowed to continue the program" without "for the next five months of training with stipends based upon need and funding available" and changing "We still recommend you save up as much as possible as stipends may not cover all your costs, and we" to "We do not recommend working during the program. We do not provide medical . . ."

File: templates/full-time-program.html [hunk 10]
```diff
@@ -18,10 +18,9 @@ <h2>
     <div class="column">
       <h1 class="blue-h1">The Program</h1>
       <p class="justify">
-        Techtonica's full-time software engineering training program is a 
-        virtual, close to year-long program that provides the opportunity for
-        women and non-binary adults seeking economic empowerment to learn technical skills in
-        a safe, inclusive space and start their first jobs in tech. We select
+        Techtonica provides the opportunity for women and non-binary adults
```
Comment (MichelleGlauser) [inline templates/full-time-program.html:21]: Sorry, could we expand this a little to "Techtonica’s full-time, virtual software engineering training program provides the opportunity . . ."

Comment (daaimah123) [reply templates/full-time-program.html]: ### ✅ Addressed in [f5a0daf](https://github.com/Techtonica/techtonica.org/pull/336/commits/f5a0daf59b8f915df48c3b6f5763dc25ed05dbd7).

File: templates/full-time-program.html (hunk 9 above)
Comment (daaimah123) [reply templates/full-time-program.html]: ### ✅ Addressed in [f5a0daf](https://github.com/Techtonica/techtonica.org/pull/336/commits/f5a0daf59b8f915df48c3b6f5763dc25ed05dbd7).

File: templates/full-time-program.html [hunk 11]
```diff
@@ -51,17 +49,39 @@ <h1 class="blue-h1">The Program</h1>
       <div class="blue-background">
         <canvas id="myChart"></canvas>
       </div>
+      <p class="justify">
+        As many companies have changed their priorities over the last few years, Techtonica 
+        has had difficulty relying on a corporate sponsorship model. As a result, Techtonica 
+        is adopting a tuition-based model with need-based tuition and stipend
```
Comment (MichelleGlauser) [inline templates/full-time-program.html:55]: "with need-based tuition" —> "with need-based, sliding-scale tuition"

Comment (daaimah123) [reply templates/full-time-program.html]: ### ✅ Addressed in [7139de1](https://github.com/Techtonica/techtonica.org/pull/336/commits/7139de1caac514b497b04a2cfc08ed0517b0da00).

**Pull request merged**